manim/mobject/types/image_mobject.py

```diff
@@ -117,14 +117,23 @@ class ImageMobjectFromCamera(AbstractImageMobject):
 
     def __init__(self, camera, **kwargs):
         self.camera = camera
+        self.pixel_array = self.camera.pixel_array
         AbstractImageMobject.__init__(self, **kwargs)
 
+    # TODO: Get rid of this.
     def get_pixel_array(self):
-        return self.camera.get_pixel_array()
+        self.pixel_array = self.camera.pixel_array
+        return self.pixel_array
 
     def add_display_frame(self, **kwargs):
         config = dict(self.default_display_frame_config)
         config.update(kwargs)
         self.display_frame = SurroundingRectangle(self, **config)
         self.add(self.display_frame)
         return self
+
+    def interpolate_color(self, mobject1, mobject2, alpha):
+        assert mobject1.pixel_array.shape == mobject2.pixel_array.shape
```

Comment:
**Suggestion :** Maybe use an exception instead of an assertion (or at  least add a descriptive message)

Reply:
I have no recollection of having changed this. I think this might pertain to @Aathish04 commit?

Reply:
> I have no recollection of having changed this. I think this might pertain to @Aathish04 commit?

Yes, I added this based off of `ImageMobject`'s `interpolate_color` method. 

I'll add a more descriptive assertion error.

Reply:
But still, I think an `Exception` is better;

Reply:
> But still, I think an Exception is better;

It now does this:

```py
AssertionError: Mobject pixel array shapes incompatible for interpolation.
Mobject 1 (ImageMobjectFromCamera) : (480, 854, 4)
Mobject 2 (ImageMobjectFromCamera) : (480, 854, 4)
```

Do you still think an Exception would be better, or is this fine?

Reply:
This is fine :D